docs/plugins/layout.md

```diff
@@ -422,13 +422,14 @@ Patch the style with `another`.
 | Return    | `self`                          |
 | Private   | This method can't be inherited. |
 
-### `__new()` {#style.\_\_new}
+### `__new(style)` {#style.\_\_new}
```

Comment:
I'm a bit fuzzy on why `ui.Style()` was made to accept a `Style` as an argument. I searched the repo and couldn't find anywhere that actually uses it that way. 

Do you know of any real-world use cases that require it? Generally, for APIs I haven't fully thought through yet, I'd like not add them to the docs to keep the possibility to change them in the future without breaking too many things

Reply:
Does

```diff
-			:style(ui.Style(th.confirm.border))
+			:style(th.confirm.border)
```

work for you?

`th.confirm.border` itself is a `Style`, there seems no point to pass it to `ui.Style()` (which accepts a `Style`) to make a new `Style`